cmd/auth-provider-gcp/BUILD

```diff
@@ -0,0 +1,25 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_binary",
+    "go_library",
+)
+load("//defs:version.bzl", "version_x_defs")
+
+go_binary(
+    name = "auth-provider-gcp",
+    embed = [":go_default_library"],
+)
+
+go_library(
+    name = "go_default_library",
+    srcs = ["main.go"],
+    importpath = "k8s.io/cloud-provider-gcp/cmd/auth-provider-gcp",
+    deps = [
+    ],
+)
+
+load("//defs:container.bzl", "image")
```

Comment:
I don't believe we're running this inside a container, so I believe this should be unnecessary.

Reply:
Done